extensions/mssql/src/mssql.d.ts

```diff
@@ -288,7 +291,9 @@ export const enum SchemaObjectType {
 	ServerAuditSpecifications = 63,
 	ServerRoleMembership = 64,
 	ServerRoles = 65,
-	ServerTriggers = 66
+	ServerTriggers = 66,
```

Comment:
the number match dacfx right :)

Reply:
Yup.  Or at least the five or six ones I randomly spot-checked do! :joy: